### DISCUSSION TITLE
Specify multiple clients

### DISCUSSION THREAD

**Discussion by Gamei666** [ltsp/ltsp#721] [Q&A]
Hello. Please advise how to specify multiple clients in ltsp.conf? Examples in documentation only show how to specify one client by mac or hostname but is it possible to specify multiple clients by hostname in one line. For example [ltsp10] [ltsp30] [ltsp20] etc? What syntax i should use?

**Comment by alkisg** [comment]
Here's one way, with INCLUDE:

```shell
[NVIDIA]
VAR1=value1...
VAR2=value2...

[ltsp10]
INCLUDE=NVIDIA

[ltsp20]
INCLUDE=NVIDIA
```

Here's another way. The section names are actually implemented using the shell's `case` command, so it can accept the case OR operator, `|`, and even wildcards:

```shell
[ltsp10 | ltsp20 | ltsp30]
VAR1=value1...
```

So since it accepts wildcards (`?` and `*`), it can also be written as:

```shell
[ltsp?0]
VAR1=value1...
```
